README.md

```diff
@@ -98,22 +121,32 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Your Answer:** 
 
+We're getting the environment variable - whether it's development or production and setting the base URL based on that. So when we launch to a live server there are fewer places we need to update our code.
+
 ---
 
-- [ ] Let's store the token in LocalStorage with a key of `journal-app`.
+- [X] Let's store the token in LocalStorage with a key of `journal-app`.
 
 * **Question:** Why are we storing the token?
 
 * **Your Answer:**
 
+So that we can use it programatically. We can get it from local storage and use it in our app.
```

Comment:
Correct, otherwise we would need to login every time the page refreshed or we tried to manually visit a route.